diagnostic.md

```diff
@@ -19,7 +27,9 @@ c = c + d;
 ```
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
-
+c == 7;
```

Comment:
I assume you mean that the value of c is equal to 7. The expression you wrote here would actually evaluate to `true`. Do you understand why?